config/features/event/core.entity_form_display.taxonomy_term.event_categories.default.yml

```diff
@@ -46,4 +46,6 @@ content:
     settings:
       display_label: true
     third_party_settings: {  }
-hidden: {  }
+hidden:
+  publish_on: true
+  unpublish_on: true
```

Comment:
I'm curious why this is different.

Reply:
taxonomy doesn't have moderation so the publish_state and unpublish_state don't apply